kafka/src/main/resources/config/base.conf

```diff
@@ -80,6 +68,7 @@ ad-manager {
   }
   inbound-topic = "mapped-metrics"
   outbound-topic = "anomalies"
+  model-service-uri-template = "http://aa.local:32300/modelservice/api/models/search/findTopByDetectorUuidOrderByBuildTimestampDesc?uuid=%s"
```

Comment:
We need to fix this long search name using `@Query`. Oh I see below that you are using `@Query` already but not renaming it. I suggest `findLatestByDetectorUuid`?

Reply:
Fixed in PR #240